settlement/bitflyer/upload.go

```diff
@@ -267,20 +253,24 @@ func setupSettlementTransactions(
 		set = append(set, aggregatedTx)
 		settlementRequests[index] = set
 	}
-	return settlements, settlementRequests, notSubmittedSettlements, nil
+	return settlements, settlementRequests, notSubmittedSettlements, numReduced, nil
 }
 
 func createBitflyerRequests(
```

Comment:
Could be me, but I think this function would benefit from a comment at the top explaining what it does.
I.e. transforms our antifraud format into requests to BF format